src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversRobots = {};
+
+  for (const key in robot) {
+    if (inversRobots.hasOwnProperty(robot[key])) {
+      return null;
+    } else {
+      inversRobots[robot[key]] = key;
+    }
```

Comment:
 Можна і без `else`, так як  'return` повертає значення і закінчує цикл
```suggestion
    } 
      inversRobots[robot[key]] = key;
    }
```